.changeset/great-cougars-buy.md

```diff
@@ -0,0 +1,6 @@
+---
+"github.com/livekit/protocol": major
+"@livekit/protocol": major
```

Comment:
same as above

Reply:
Do I just run the command again?

Reply:
Oops, missed the earlier comment. Will manually edit.